src/styles/global/layout.scss

```diff
@@ -49,15 +49,6 @@ main,
   padding-left: var(--fs-grid-padding);
 }
 
-// Avoids double scroll issue on the page when overlay is open
-#layout.no-scroll {
```

Comment:
Nicee! I'm touching this piece of code in another PR but I can sync this change there 👀